### PR TITLE
SNET-870 Unable to click continue button even channel balance is available for paid calls

### PR DESCRIPTION
**Pull request by rajeevbbqq**

### CODE REVIEW

File: src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/ExpiredSession/MetamaskFlow/index.js [hunk 1]
```diff
@@ -207,7 +207,10 @@ class MetamaskFlow extends Component {
     return this.PaymentInfoCardData.find(el => el.title === "Channel Balance").value;
   };
 
-  shouldContinueBeEnabled = () => this.state.mpeBal > 0 && this.props.isServiceAvailable;
+  shouldContinueBeEnabled = () => {
+    const { channelBalance, totalPrice } = this.state;
+    return channelBalance >= totalPrice;
```
Comment (Vivek205) [inline src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/ExpiredSession/MetamaskFlow/index.js]: where are we setting the channelBalance?
Actually, mpeBal should be same as channelBalance.

File: src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/ExpiredSession/MetamaskFlow/index.js [hunk 2]
```diff
@@ -207,7 +207,10 @@ class MetamaskFlow extends Component {
     return this.PaymentInfoCardData.find(el => el.title === "Channel Balance").value;
   };
 
-  shouldContinueBeEnabled = () => this.state.mpeBal > 0 && this.props.isServiceAvailable;
+  shouldContinueBeEnabled = () => {
+    const { mpeBal, totalPrice } = this.state;
+    return mpeBal >= totalPrice;
```
Comment (Vivek205) [inline src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/ExpiredSession/MetamaskFlow/index.js]: this.props.isServiceAvailable; should also be considered..
--
And what happens if there is no balance in mpe but there is enough balance in channel?
Will continue button be enabled?

Comment (rajeevbbqq) [reply src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/ExpiredSession/MetamaskFlow/index.js]: Fix in the latest commit

File: src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/ExpiredSession/MetamaskFlow/index.js (hunk 1 above)
Comment (rajeevbbqq) [reply src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/ExpiredSession/MetamaskFlow/index.js]: After fetching available balance from channel we are setting channel balance to local variable

**Review by Vivek205 — APPROVED** [review]

**Pull request merged**